policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -225,14 +230,69 @@ impl kubert::index::IndexNamespacedResource<Service> for Index {
 
     fn delete(&mut self, namespace: String, name: String) {
         tracing::debug!(name, namespace, "deleting service");
-        let service_ref = ServiceRef { name, namespace };
+        let service_ref = ResourceRef { name, namespace };
         self.service_info.remove(&service_ref);
         self.services_by_ip.retain(|_, v| *v != service_ref);
 
         self.reindex_services()
     }
 }
 
+impl kubert::index::IndexNamespacedResource<linkerd_k8s_api::UnmeshedNetwork> for Index {
+    fn apply(&mut self, u: linkerd_k8s_api::UnmeshedNetwork) {
+        let um: UnmeshedNetwork = u.into();
+        tracing::debug!(um.name, um.namespace, "indexing unmeshed network");
+
+        self.unmeshed_networks.insert(
+            ResourceRef {
+                name: um.name.clone(),
+                namespace: um.namespace.clone(),
+            },
+            um,
+        );
+    }
+
+    fn delete(&mut self, namespace: String, name: String) {
+        tracing::debug!(name, namespace, "deleting unmeshed networks");
+        let um_ref = ResourceRef { name, namespace };
+        self.unmeshed_networks.remove(&um_ref);
+    }
+}
+
+impl kubert::index::IndexNamespacedResource<Pod> for Index {
+    fn apply(&mut self, pod: Pod) {
+        let ns = pod.namespace().unwrap();
+        let name = pod.name_unchecked();
+        tracing::debug!(name, ns, "indexing pod");
```

Comment:
it'd be nice to call this field `namespace`, rather than `ns`. it'll be friendlier to readers

Reply:
I'm fairly sure that we use 'ns' is commonly used throughout the codebase.

There isn's really ambiguity to readers at debug-level--we expect you to have context about internals here. Furthermore, 'ns' is a fairly common abbreviation (it is in fact the canonical abbreviation for 'Namespace' resources -- `k get ns`).

Reply:
marking this as resolved, thank you for the context :white_check_mark: